packages/terra-functional-testing/src/terra-cli/wdio/index.js

```diff
@@ -30,32 +99,43 @@ const cli = {
       locales: {
         type: 'array',
         describe: 'A list of language locales for the test run.',
-        default: ['en'],
-      },
-      themes: {
-        type: 'array',
-        describe: 'A list of themes for the test run.',
-        default: ['terra-default-theme'],
+        default: () => {
+          if (process.env.LOCALE) {
+            return [process.env.LOCALE];
+          }
+
+          return ['en'];
+        },
       },
-      hostname: {
+      site: {
         type: 'string',
-        describe: 'Automation driver host address.',
-      },
-      port: {
-        type: 'number',
-        describe: 'Automation driver port.',
+        describe: 'A file path to a static directory of assets. When defined, an express server will launch to serve the assets and disable running webpack.',
+        default: () => {
+          if (process.env.SITE) {
+            return [process.env.SITE];
```

Comment:
does the site value need to be inside an array?

Reply:
Yes, it should be a string. Good catch! https://github.com/cerner/terra-toolkit/pull/583/commits/b0d401b05c22bf675b0eaf05d09b897f3c9e76df